tests/foreman/cli/test_discoveredhost.py

```diff
@@ -574,79 +345,9 @@ def test_positive_provision_pxe_host_with_uefi_grub2(self):
         :CaseImportance: High
         """
 
-    @pytest.mark.stubbed
     @pytest.mark.tier3
-    def test_positive_provision_pxe_host_with_uefi_grub2_sb(self):
-        """Provision the pxe-based UEFI Secureboot discovered host from cli
-        using PXEGRUB2 loader
-
-        :id: 285ebbac-934b-45ae-a0ca-0dc450836428
-
-        :Setup:
-            1. Create an UEFI SB VM and set it to boot from a network
-            2. Synchronize RHEL6 kickstart repo (rhel7 kernel too new for GRUB)
-            3. for getting more detailed info from FDI, remaster the image to
-               have ssh enabled
-
-        :steps:
-            1. Build a default PXE template
-            2. Run assertion step #1
-            3. Boot the VM (from NW)
-            4. Run assertion steps #2-4
-            5. Provision the discovered host
-            6. Run assertion steps #5-9
-
-        :expectedresults: Host should be provisioned successfully
-            1. Ensure the tftpboot files are updated
-
-              1.1 Ensure fdi-image files have been placed under tftpboot/boot/
-              1.2 Ensure the 'default' pxelinux config has been placed under
-              tftpboot/pxelinux.cfg/
-              1.3 Ensure the discovery section exists inside pxelinux config,
-              it leads to the FDI kernel and the ONTIMEOUT is set to discovery
-
-            2. Ensure PXE handoff goes as expected (tcpdump -p tftp)
-            3. Ensure FDI loaded and successfully sent out facts
-
-                3.1 ping vm
-                3.2 ssh to the VM and read the logs (if ssh enabled)
-                3.3 optionally sniff the HTTP traffic coming from the host
-
-            4. Ensure host appeared in Discovered Hosts on satellite
-            5. Ensure the tftpboot files are updated for the hosts mac
-            6. Ensure PXE handoff goes as expected (tcpdump -p tftp)
-            7. Optionally ensure anaconda loaded and the installation finished
-            8. Ensure the host is provisioned with correct attributes
-            9. Ensure the entry from discovered host list disappeared
-
-        :CaseAutomation: NotAutomated
-
-        :CaseImportance: High
-        """
-
-    @pytest.mark.tier3
-    def test_positive_delete_pxeless_host(self):
```

Comment:
There's no difference between removing a pxeless host or a pxe-based host - it's a discovered host entity in foreman.